src/webapp/azext_webapp/custom.py

```diff
@@ -312,73 +312,199 @@ def _should_create_new_asp(cmd, rg_name, asp_name, location):
     return True
 
 
-def enable_one_deploy(cmd, resource_group_name, name, src, deploy_type=None, is_async=None, target_path=None, timeout=None, slot=None):
-    import ntpath
-    logger.info("Preparing for deployment")
-    user_name, password = _get_site_credential(cmd.cli_ctx, resource_group_name, name, slot)
+# OneDeploy
+def perform_onedeploy(cmd,
+                      resource_group_name,
+                      name,
+                      src_path=None,
+                      src_url=None,
+                      type=None,
+                      async=None,
```

Comment:
"async" is a reserved keyword in python. You should not use it.

Reply:
Did it pass CI?

Reply:
from https://github.com/Azure/azure-cli-extensions/pull/2768/checks, it passed all CI tasks

Reply:
There is a new PR open which fixes this - https://github.com/Azure/azure-cli-extensions/pull/2811

Reply:
I tried in python37
```
In [5]: def f(async=1):
   ...:     print(async)
  File "<ipython-input-5-6af617185783>", line 1
    def f(async=1):
              ^
SyntaxError: invalid syntax
```